resources/migrations/20220120121117-user-mappings.up.sql

```diff
@@ -0,0 +1,5 @@
+CREATE TABLE user_mappings (
+  id serial NOT NULL PRIMARY KEY,
+  usermappingsdata jsonb
```

Comment:
yeah true - i was thinking that since we have already used the blob elsewhere that it would be a natural fit here too. but OTOH there is nothing else than the blob so it could be either this way, or as you wrote, 3 column table with indexing. the DB version could be a better fit here

Reply:
used indexed table instead of cache solution, which i think simplifies the solution and makes it more readable